src/app.es6.js

```diff
@@ -96,19 +95,13 @@ class App {
   error (err, ctx, app) {
     var status = err.status || 500;
     var message = err.message || 'Unkown error';
-
-    var reroute = '/' + status;
     var url = '/' + status;
 
-    var query = querystring.stringify({
-      originalUrl: ctx.request.url || '/',
-    });
-
-    url += '?' + query;
```

Comment:
changing the transient errors to simply have a "reload the page" button and 404s still show "view on reddit".  this seemed unnecessary as a result.